include/pros/rotation.h

```diff
@@ -0,0 +1,186 @@
+/**
+ * \file pros/rotation.h
+ *
+ * Contains prototypes for functions related to the VEX Rotation sensor.
+ *
+ * Visit https://pros.cs.purdue.edu/v5/tutorials/topical/rotation.html to learn
+ * more.
+ *
+ * This file should not be modified by users, since it gets replaced whenever
+ * a kernel upgrade occurs.
+ *
+ * Copyright (c) 2017-2020, Purdue University ACM SIGBots.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#ifndef _PROS_ROTATION_H_
+#define _PROS_ROTATION_H_
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+namespace pros {
+namespace c {
+#endif
+
+/**
+ * Resets Rotational Sensor 
+ *
+ * Resets rotation sensor by multiplying the rotation reading by 
+ * -1 only if the direction was recently reversed.
```

Comment:
at a glance i'm not really sure what this means

Reply:
We were talking about this a bit on discord. From what will was saying was that it was cursed and we might need a custom implementation for now.

Reply:
(Reflected from Discord) to make up for the cursed reset function, we also now have rotation_reset_position which resets the position to 0.